### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by oleksandranahorniuk**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -7,4 +7,50 @@ describe('getHumanAge', () => {
...
+    expect(getHumanAge(28, 28))
+      .toEqual([3, 2]);
+  });
+
+  test('return 2 + Math.floor((animalAge - first - second) / other)'
+  + 'if age of cat and dog more than 24', () => {
+    expect(getHumanAge(100, 100))
+      .toEqual([21, 17]);
+  });
 });
```
Comment (oablokhina) [inline src/getHumanAge.test.js:56]: add tests for invalid data (symbols, negative numbers)
